Sources/ContainerizationOCI/Spec.swift

```diff
@@ -313,8 +313,8 @@ public struct Mount: Codable, Sendable {
         source: String = "",
         destination: String,
         options: [String] = [],
-        uidMappings: [LinuxIDMapping] = [],
-        gidMappings: [LinuxIDMapping] = []
+        uidMappings: [LinuxIDMapping]? = nil,
```

Comment:
Side note: In case we ever want to take a crack at generating Swift from spec, https://github.pie.apple.com/vessel/swift-docker-api demonstrates an approach for converting Open API/Swagger to Swift bindings with some stuff to handle things that are optional in practice but required according to what the spec says.